src/control/destination/mod.rs

```diff
@@ -24,30 +24,32 @@
 //! - We need some means to limit the number of endpoints that can be returned for a
 //!   single resolution so that `control::Cache` is not effectively unbounded.
 
-use indexmap::IndexMap;
-use std::sync::{Arc, Weak};
-use std::time::Duration;
-
 use futures::{
+    future,
     sync::mpsc,
     Async,
     Future,
     Poll,
     Stream
 };
+use indexmap::IndexMap;
+use std::fmt;
+use std::sync::{Arc, Weak};
```

Comment:
It looks like some imports were moved around, was this a `rustfmt` change?

Reply:
probably, sort of. i've tried to make our imports more uniform as i touch files, since they're all over the place.